app/pdfs/passengers_PDF.rb

```diff
@@ -0,0 +1,34 @@
+require 'prawn/table'
+
+class PassengersPDF < Prawn::Document
```

Comment:
This filename is incorrect for autoload to work on non-Macs.

Rails expects this class to be in a file named `passengers_pdf.rb` (or `passengers_p_d_f.rb`, but don't) Mac has a dumb, case-insensitive filesystem, which makes it work, but this won't work in production.